examples/jsm/loaders/LUTImageLoader.js

```diff
@@ -0,0 +1,160 @@
+// http://download.autodesk.com/us/systemdocs/help/2011/lustre/index.html?url=./files/WSc4e151a45a3b785a24c3d9a411df9298473-7ffd.htm,topicNumber=d0e9492
+// https://community.foundry.com/discuss/topic/103636/format-spec-for-3dl?mode=Post&postID=895258
+import {
+	Loader,
+	TextureLoader,
+	DataTexture,
+	Data3DTexture,
+	RGBAFormat,
+	UnsignedByteType,
+	ClampToEdgeWrapping,
+	LinearFilter,
+} from 'three';
+
+export class LUTImageLoader extends Loader {
+
+	constructor( flipVertical = true ) {
+
+		super();
+
+		this.flip = flipVertical;
+
+	}
+
+	load( url, onLoad, onProgress, onError ) {
+
+		const loader = new TextureLoader( this.manager );
+
+		loader.setPath( this.path );
+		loader.load( url, texture => {
```

Comment:
@Mugen87 correct me if I'm wrong but do we need to set other Loader fields here like "crossOrigin", etc?

Reply:
Yes, indeed. Good catch! At least the following line should be added.
```js
loader.setCrossOrigin( this.crossOrigin );
```
Others like `setRequestHeader()` or `setWithCredentials()` are not relevant for `TextureLoader`.

Reply:
Added